src/sinks/influxdb/logs.rs

```diff
@@ -166,12 +166,7 @@ impl HttpSink for InfluxDbLogsSink {
         let mut event = event.into_log();
 
         // Measurement
-        let name = "vector";
-        let measurement = encode_namespace(
-            Some(self.namespace.as_str()).filter(|namespace| !namespace.is_empty()),
-            '.',
-            name,
-        );
+        let measurement = self.measurement.clone();
```

Comment:
FYI I just merged a change to the influxdb sinks (#10109) that makes this clone unnecessary as it can be passed in to `influx_line_protocol` by reference.